packages/react/package.json

```diff
@@ -55,7 +55,7 @@
   },
   "devDependencies": {
     "@faker-js/faker": "^8.0.2",
-    "@slashid/slashid": "^3.9.0",
+    "@slashid/slashid": "link:/Users/jake/dev/ng-evangelion/web/sdk",
```

Comment:
Leaving a note here so we don't forget replacing this with the actual package :)